lib/rules/length-zero-no-unit/README.md

```diff
@@ -87,3 +87,25 @@ a { top: var(--foo, 0px); }
 ```css
 a { top: --bar(0px); }
 ```
+
+### `ignoreAtRulePreludes: ["/regex/", /regex/, "string"]`
+
+Ignore units for zero lengths within the specified at-rule preludes.
+
+Given:
+
+```json
+["function", "/^mix/"]
```

Comment:
```suggestion
["--foo", "/^--bar/"]
```

For "only use standard CSS syntax in examples and options" [convention](https://stylelint.io/developer-guide/rules#write-the-readme).